site/docs/components/stepper.mdx

```diff
@@ -878,82 +868,73 @@ The HDS stepper offers multiple predefined states for steps to be used.
 - `disabled` state for steps that are not available yet (e.g. require a previous step to be completed first)
 
 <Playground>
-<div>
-  <div>
-    <Stepper
-      className="stepper-states-example"
-      language="en"
-      onStepClick={function noRefCheck(){}}
-      selectedStep={1}
-      steps={[
-        {
-          label: 'Completed',
-          state: StepState.completed
-        },
-        {
-          label: 'Current',
-          state: StepState.available
-        },
-        {
-          label: 'Available',
-          state: StepState.available
-        },
-        {
-          label: 'Needs attention',
-          state: StepState.attention
-        },
-        {
-          label: 'Paused',
-          state: StepState.paused
-        },
-        {
-          label: 'Disabled',
-          state: StepState.disabled
-        }
-      ]}
-    />
-  </div>
-</div>
+  <Stepper
+    language="en"
+    onStepClick={function noRefCheck(){}}
+    selectedStep={1}
+    steps={[
+      {
+        label: 'Completed',
+        state: StepState.completed
+      },
+      {
+        label: 'Current',
+        state: StepState.available
+      },
+      {
+        label: 'Available',
+        state: StepState.available
+      },
+      {
+        label: 'Needs attention',
+        state: StepState.attention
+      },
+      {
+        label: 'Paused',
+        state: StepState.paused
+      },
+      {
+        label: 'Disabled',
+        state: StepState.disabled
+      }
+    ]}
+  />
 </Playground>
 
 #### React code example:
 ```tsx
-<div>
-  <div>
-    <Stepper
-      className="stepper-states-example"
-      language="en"
-      onStepClick={function noRefCheck(){}}
-      selectedStep={1}
-      steps={[
-        {
-          label: 'Completed',
-          state: StepState.completed
-        },
-        {
-          label: 'Current',
-          state: StepState.available
-        },
-        {
-          label: 'Available',
-          state: StepState.available
-        },
-        {
-          label: 'Needs attention',
-          state: StepState.attention
-        },
-        {
-          label: 'Paused',
-          state: StepState.paused
-        },
-        {
-          label: 'Disabled',
-          state: StepState.disabled
-        }
-      ]}
-    />
-  </div>
-</div>
+<Stepper
+  className="stepper-states-example"
```

Comment:
I guess this className could be omitted too

Reply:
- [ ] Omit className